snapcraft/internal/parser.py

```diff
@@ -358,4 +359,4 @@ def _write_parts_list(path, master_parts_list):
 
 
 if __name__ == '__main__':  # pragma: no cover
-    main()                  # pragma: no cover
+    sys.exit(main())        # pragma: no cover
```

Comment:
Good catch.  You're right we want 1 or 0, maybe another in the future for things like network issues getting the index.